content/en/internal/configuration-database/adding-new-projects/_index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Options for adding a new Project in GocDB"
+weight: 30
+description: >-
+     How to create a new Project in GocDB.
+---
+
+## Introduction
+
+GocDB is multi-tenanted; it can host multiple projects in the same instance.
+There are a number of different deployment scenarios that can be used to support
+new projects detailed below. Please contact the GocDB admins/EGI Operations to
+request the addition of a new project.
+
+## 1) Add resources (sites/services) to an existing project
+
+- Resources (NGIs, Sites, Services) would be hosted under an existing project,
+e.g. the ‘EGI’ project.
+- The new resources would be subject to the rules/roles of the existing project,
+such as site certification status changes and project controlled user memberships.
+- The resources could not be filtered using a custom scope tag.
+
+<!-- markdownlint-disable no-inline-html -->
+## 2) Add resources (sites/services) to an existing project and add a new Scope tag to represent a sub-grouping
+<!-- markdownlint-enable no-inline-html -->
+
+- Resources would be hosted under an existing project, and a new scope tag would
+be added for the purposes of resource filtering.
+- Since the resources are still hosted under an existing project, the resources
+would still be subject to the rules/roles of that project such as project
+controller user memberships.
```

Comment:
```suggestion
- Since the resources are still hosted under an existing project, the new resources
would still be subject to the rules/roles of the existing project, such as site 
certification status changes and project controlled user memberships.
```